src/app/static/src/js/components/runWorkflow/runWorkflow.vue

```diff
@@ -76,12 +101,24 @@ export default Vue.extend<Data, Methods, unknown, unknown>({
             this.disableRename = false
         },
         handleComplete: function () {
-            //handle submitted report here
+            api.post(`/workflow`, this.workflowMetadata)
```

Comment:
This doesn't look quite right - `workflowMetadata` doesn't seem to include `instances`, `git_branch` or `git_commit` - which need to be sent to `/workflow`. In other words, why do we have a new type `WorkflowMetadata` when we already have `RunWorkflowMetadata`?

On the other hand, when I re-run a workflow it's failing because it's sending extra information in the request: the reports have old keys and workflow keys pre-populated. I can't see why this is happening but it is a problem.

Reply:
Yes, the runWorkflowMetadata from the wizard ought to be runnable. 

I think the issue with re-running is a bug in runWorkflowCreate - it's  trying to create a runnable workflow from the old workflow metadata, but isn't stripping the keys from `reports`. https://github.com/vimc/orderly-web/blob/master/src/app/static/src/js/components/runWorkflow/runWorkflowCreate.vue#L119 
Similar issue with Clone I assume.

Reply:
I guess I just assumed that since we were only sending name, changelog, and reports with the fetch command that the same would apply here.

Reply:
> I think the issue with re-running is a bug in runWorkflowCreate - it's trying to create a runnable workflow from the old workflow metadata, but isn't stripping the keys from `reports`. https://github.com/vimc/orderly-web/blob/master/src/app/static/src/js/components/runWorkflow/runWorkflowCreate.vue#L119
> Similar issue with Clone I assume.

Aha. That's out of scope for this ticket so I have created a new one: mrc-2558